frontends/qt/main.cpp

```diff
@@ -32,6 +32,35 @@ static WebClass* webClass;
 static QMutex webClassMutex;
 static QSystemTrayIcon* trayIcon;
 
+class BitBoxApp : public QApplication
+{
+public:
+    BitBoxApp(int &argc, char **argv): QApplication(argc, argv)
+    {
+    }
+
+#if defined(Q_OS_MACOS)
+    bool event(QEvent *event) override
+    {
+        if (event->type() == QEvent::FileOpen) {
+            QFileOpenEvent* openEvent = static_cast<QFileOpenEvent*>(event);
+            if (!openEvent->url().isEmpty()) {
```

Comment:
was a bit worried `openEvent->url()...` could generate a segfault but from https://github.com/qt/qtbase/blob/31f90e9/src/gui/kernel/qevent.h#L927-L943 and https://github.com/qt/qtbase/blob/31f90e9/src/gui/kernel/qevent.cpp#L3615-L3626 it seems QFileOpenEvent's `m_url` is always valid :+1: